2015/proposals/girdhar-yashasvi.md

```diff
@@ -0,0 +1,247 @@
+#Result-aggregation server for the installation-test scripts
+
+##Personal Details
+
+* Name: Yashasvi Girdhar
+* Email: yash.girdhar@gmail.com
+* Personal Webpage : http://about.me/yashgirdhar
+* Github profile : https://github.com/itsyash
+* IRC nick (on freenode) : yash_
+* Telephone: +91-8143914414
```

Comment:
You can remove you phone number, check a19315a. If you could squash your commits for this information not be under the version control will be great.